en/docs/reference/connectors/ceridiandayforce-connector/ceridiandayforce-connector-example.md

```diff
@@ -22,86 +22,227 @@ parameters must be passed in the body as we will see below.
+    | xRefCode           | json-eval($.xRefCode)             |
+
+    You can add these properties by clicking on the **Add Parameter** of the **Property Group** tab. When entering the `Property Value` fields, make sure to select the `Ex` button to specify the value types as expressions.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/dayforce/add-parameters-to-property-group.png" title="Add Property Group Mediator" width="800" alt="Add Property Group Mediator"/>
+
+6. Now, add the **Ceridian Dayforce** connector to the mediation logic. Click on `+` in the graphical view and select the `cerdiandayforce` connector from the list of connectors. Select the `init` operation from the list of operations.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/dayforce/add-dayforce-init.png" title="Add Connector" width="800" alt="Add Connector"/>
 
-Now we can export the imported connector and the API into a single CAR application. CAR application is the one we are 
-going to deploy to server runtime. 
+7. Click on **Add Parameter** and add the following parameters to the Ceridian Dayforce connector's `init` operation.
 
-{!includes/reference/connectors/exporting-artifacts.md!}
+    | Parameter name | Parameter value (as expression) |
+    |----------------|---------------------------------|
+    | username       | $ctx:username                   |
+    | password       | $ctx:password                   |
+    | clientNamespace | $ctx:clientNamespace           |
+    | apiVersion     | $ctx:apiVersion                 |
 
-Now the exported CApp can be deployed in the integration runtime so that we can run it and test.
+    <img src="{{base_path}}/assets/img/integrate/connectors/dayforce/add-dayforce-init-param.png" title="Add Parameters to Dayforce Init" width="800" alt="Add Parameters to Dayforce Init"/>
+
+    Click on **Submit** to save the configurations.
+
+8. Add the `getEmployeeAddresses` operation to the mediation logic. Click on `+` in the graphical view and select the `cerdiandayforce` connector from the list of connectors. Select the `getEmployeeAddresses` operation from the list of operations.
+
+    <img src="{{base_path}}/assets/img/integrate/connectors/dayforce/add-dayforce-get-addresses.png" title="Add Get Employee Addresses" width="800" alt="Add Get Employee Addresses"/>  
+
+9. Click on the **Add Parameter** and add the following parameters to the Ceridian Dayforce connector's `getEmployeeAddresses` operation.
+
+    | Parameter name | Parameter value (as expression) |
+    |----------------|---------------------------------|
+    | xRefCode       | $ctx:xRefCode                   |
+    | contextDate    | $ctx:contextDate                |
+    | contextDateRangeFrom | $ctx:contextDateRangeFrom   |
+    | contextDateRangeTo | $ctx:contextDateRangeTo       |
+
+    Click on **Submit** to save the configurations.
+
+10. Add a **Send** mediator to the mediation logic to send the response back to the client.
```

Comment:
```suggestion
10. Add a **Respond** mediator to the mediation logic to send the response back to the client.
```

Reply:
You can point to the Respond mediator docs in this space here.

[Respond Mediator]({{base_path}}/reference/mediators/respond-mediator/)